gcsio/src/test/java/com/google/cloud/hadoop/gcsio/integration/GoogleCloudStorageImplTest.java

```diff
@@ -567,4 +576,43 @@ private static List<String> getExpectedRequestsForCreateObject(
                 .collect(toList()))
         .build();
   }
+
+  @Test
+  public void initializeRequest_withGcsAuditLogEnabled() throws IOException {
+    AssertingLogHandler jsonLogHander = new AssertingLogHandler();
+    Logger jsonTracingLogger =
+        jsonLogHander.getLoggerForClass(EventLoggingHttpRequestInitializer.class.getName());
+
+    try {
+
+      String ugiUser = UUID.randomUUID().toString();
+      UserGroupInformation ugi = UserGroupInformation.createRemoteUser(ugiUser);
```

Comment:
where will this create the user?

Reply:
Sets the kerberos user for this process.